README.md

```
@@ -1,2 +1,27 @@
-# AIBE1-Project2-Team03
-AIBE1 TEAM03 호식이세마리 | 삼식이(병아리, 3세)와 떠나는 프로젝트 대모험
+# TEAMO !
+> AIBE1 TEAM03 호식이세마리 | 팀원 준비 완료! 팀워크가 필요할 땐, Teamo
+
+
+
+### ✅ application-dev.yml
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)